tests/lib/schema_validation_test.py

```diff
@@ -162,3 +191,158 @@ def test_invalid_state(self, default_data):
         with pytest.raises(js.ValidationError) as err:
             libnmstate.validator.validate(default_data)
         assert 'bad-state' in err.value.args[0]
+
+
+class TestVlanFiltering(object):
+    def test_basic_configuration(self, default_data):
+        libnmstate.validator.validate(default_data)
+
+    def test_bridge_vlan_filtering_only_values_on_schema(self, default_data):
+        bridge_state = self._get_bridge_iface_state(default_data[INTERFACES])
+        invalid_vlan_filtering_state = self._get_bridge_vlan_filtering_state(
+            vlan_filtering_enabled=True, invalid_key='ka-boom!'
+        )
+        bridge_state[LB.CONFIG_SUBTREE][
+            LB.OPTIONS_SUBTREE
+        ] = invalid_vlan_filtering_state
+        with pytest.raises(js.ValidationError) as err:
+            libnmstate.validator.validate(default_data)
+        assert '(\'invalid_key\' was unexpected)' in err.value.args[0]
```

Comment:
PEP8 tells us to keep uniform `'` or `"` with one exception: It is better to use the other quotation mark than to escape. Could you please change it to `"('invalid_key' was unexpected)"`